src/git/repo.rs

```diff
@@ -608,6 +664,51 @@ Either create it, or update the main branch setting by running:
         }
     }
 
+    /// Returns the current status of the repo index and working copy.
+    pub fn get_status(&self, git_run_info: &GitRunInfo) -> eyre::Result<Vec<StatusEntry>> {
+        let output = git_run_info.run_silent(
+            self,
+            None,
```

Comment:
Pass the event transaction ID from the caller through this parameter as well. (It shouldn't really be relevant for now, since this operation shouldn't affect the logical state of the repository, but maybe one day we'll use this information for profiling or something?)

Reply:
Done